package-lock.json

```diff
@@ -13,35 +13,46 @@
 				"xterm-addon-fit": "^0.5.0"
 			}
 		},
-		"node_modules/crypto-js": {
-			"version": "4.1.1",
-			"resolved": "https://registry.npmjs.org/crypto-js/-/crypto-js-4.1.1.tgz",
-			"integrity": "sha512-o2JlM7ydqd3Qk9CA0L4NL6mTzU2sdx96a+oOfPu8Mkl/PK51vSyoi8/rQ8NknZtk44vq15lmhAj9CIAGwgeWKw=="
-		},
-		"node_modules/esptool-js": {
+		"../esptool-js": {
 			"version": "0.2.0",
-			"resolved": "https://registry.npmjs.org/esptool-js/-/esptool-js-0.2.0.tgz",
-			"integrity": "sha512-TZehTn14FJ6YLfKc+YJlZxhcs5XZasE+85LFbju+TGtb/vb7bXUEkioCkVRUfSnmGBP3uZqAmd2/m7jpoSwOPg==",
 			"dependencies": {
 				"pako": "^2.1.0",
 				"tslib": "^2.4.1"
+			},
+			"devDependencies": {
+				"@rollup/plugin-babel": "^6.0.2",
+				"@rollup/plugin-commonjs": "^23.0.2",
+				"@rollup/plugin-json": "^6.0.0",
+				"@rollup/plugin-node-resolve": "^15.0.1",
+				"@rollup/plugin-terser": "^0.1.0",
+				"@types/pako": "^2.0.0",
+				"@types/w3c-web-serial": "^1.0.3",
+				"@typescript-eslint/eslint-plugin": "^5.43.0",
+				"@typescript-eslint/parser": "^5.43.0",
+				"babel-loader": "^9.1.0",
+				"eslint": "^8.28.0",
+				"eslint-config-prettier": "^8.5.0",
+				"eslint-plugin-prettier": "^4.2.1",
+				"prettier": "^2.7.1",
+				"rimraf": "^3.0.2",
+				"rollup": "^3.3.0",
+				"typescript": "^4.8.4"
 			}
 		},
-		"node_modules/pako": {
-			"version": "2.1.0",
-			"resolved": "https://registry.npmjs.org/pako/-/pako-2.1.0.tgz",
-			"integrity": "sha512-w+eufiZ1WuJYgPXbV/PO3NCMEc3xqylkKHzp8bxp1uW4qaSNQUkwmLLEc3kKsfz8lpV1F8Ht3U1Cm+9Srog2ug=="
+		"node_modules/crypto-js": {
+			"version": "4.1.1",
+			"resolved": "https://registry.npmjs.org/crypto-js/-/crypto-js-4.1.1.tgz",
+			"integrity": "sha512-o2JlM7ydqd3Qk9CA0L4NL6mTzU2sdx96a+oOfPu8Mkl/PK51vSyoi8/rQ8NknZtk44vq15lmhAj9CIAGwgeWKw=="
+		},
+		"node_modules/esptool-js": {
+			"resolved": "../esptool-js",
```

Comment:
Seems like this might be a local change?

Reply:
Thanks, I have fixed it now.